ganalytics.cpp

```diff
@@ -595,6 +630,10 @@ void GAnalytics::sendAppView(const QString &screenName)
     }
     query.addQueryItem("an", d->appName);
     query.addQueryItem("av", d->appVersion);
+    for (auto &key : customValues.keys())
```

Comment:
Please use a QHashIterator here. C++11 is nice but we don't know if the user of this library has it enabled.